mulighetsrommet-api/src/main/resources/application-local.yaml

```diff
@@ -77,3 +77,8 @@ app:
 
   norg2:
     baseUrl: https://norg2.dev-fss-pub.nais.io/norg2/api/v1
+
+  slack:
+    token: ${SLACK_DEV_TOKEN}
```

Comment:
Kan jeg sette en default value her? Tom streng feks.? Så slipper vi å måtte sette en miljøvariabel når vi kjører lokalt. Jeg husker ikke syntaksen for default values....

Reply:
EVt gjøre token optional, men kaste en error hvis enable: true?

Reply:
Syntax er noe ala ${TOKEN:fallback} tror jeg

Reply:
Syntaksen er `${TOKEN:-fallback}` fant jeg ut 👍